samples/Add Voice Calling/Web/JavaScript/src/MakeCall/AddParticipantPopover.js

```diff
@@ -0,0 +1,55 @@
+// © Microsoft Corporation. All rights reserved.
+
+import React, { useState, useRef } from "react";
+import { OverlayTrigger, Popover } from "react-bootstrap";
+import { Button, TextField } from 'office-ui-fabric-react'
+import { CommunicationUser } from '@azure/communication-calling';
+
+export default function AddParticipantPopover(props) {
+    const [userId, setUserId] = useState('');
+
+    function handleAddCommunicationUser() {
+        console.log('handleAddCommunicationUser', userId);
+        try {
+            props.call.addParticipant({ communicationUserId: userId });
+            window.document.body.click();
```

Comment:
this is hacky? what's the issue here? shouldn't need try catch etc.. and that fake click is not needed?